crates/bevy_hierarchy/src/child_builder.rs

```diff
@@ -309,6 +352,10 @@ pub trait BuildChildren {
     /// will have those children removed from its list. Removing all children from a parent causes its
     /// [`Children`] component to be removed from the entity.
     fn add_child(&mut self, child: Entity) -> &mut Self;
+    /// Clear children
```

Comment:
```suggestion
    /// Removes all children from this entity. The [`Children`] component will be removed if it exists, otherwise this does nothing.
```